### PR TITLE
Issue 827 mnemonic error feedback

### LINKED ISSUE #827: Bug: App doesn't show error message when mnemonic has been entered incorrectly
It seems like the app doesn’t give any response when there is an error in the mnemonic seed field (no error message)

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description

Fixes: https://github.com/litentry/litentry-app/issues/827

https://user-images.githubusercontent.com/16683923/151958235-c59f2555-0e41-4e30-a3ff-377b21a909a3.mp4

### CODE REVIEW

File: src/ui/screens/ImportAccountScreen.tsx [hunk 1]
```diff
@@ -50,49 +49,36 @@ function ImportAccount({navigation}: {navigation: NavigationProp<AccountsStackPa
...
+      encoded,
+      meta: {
+        name: account.title,
+        network: currentNetwork.key,
+        isFavorite: false,
+      },
+      isExternal: false,
+    };
+    addAccount(newAcc);
+    SecureKeychain.setPasswordByServiceId(account.password, 'BIOMETRICS', _address);
```
Comment (ishrak-imam) [inline src/ui/screens/ImportAccountScreen.tsx:80]: @franjs 
I thought we were using the formatted address in the secure key chain for import account, no?

Comment (ishrak-imam) [inline src/ui/screens/ImportAccountScreen.tsx:82]: @franjs 
How about we use the formatted address for SecureKeyChain as well. I am testing the impact of this change in a device build. Will revert if it introduces bugs.

Comment (silva-fj) [reply src/ui/screens/ImportAccountScreen.tsx:82]: Yep, this should be fine 👍🏼

File: src/hooks/useKeyboard.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      setIsVisible(false);
+    });
+
+    return () => {
+      keyboardDidShow.remove();
+      keyboardDidHide.remove();
+    };
+  }, []);
+
+  return isVisible;
```
Comment (silva-fj) [inline src/hooks/useKeyboard.ts]: To make it easier to understand what this hook does we could return `{ isVisible }`, or call the hook `useKeyboardStatus` and return `'visible' | 'hidden'`

**Review by silva-fj — APPROVED** [review]

**Pull request merged**